modules/apps/29-fee/keeper/msg_server.go

```diff
@@ -57,6 +61,10 @@ func (k Keeper) PayPacketFee(goCtx context.Context, msg *types.MsgPayPacketFee)
 func (k Keeper) PayPacketFeeAsync(goCtx context.Context, msg *types.MsgPayPacketFeeAsync) (*types.MsgPayPacketFeeAsyncResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	if k.IsLocked(ctx) {
+		return nil, types.ErrFeeModuleLocked
+	}
+
 	if err := k.EscrowPacketFee(ctx, msg.PacketId, msg.PacketFee); err != nil {
```

Comment:
Should we move the `IsFeeEnabled` check from EscrowPacketFee to here and make EscrowPacketFee private?